Bottom: cd48268efc94b52ce52e38716814999c1eeab0a8
Top:    ea0966ba881105d23359b60e14575ff763c9549f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:13 +0000

Export multi-body parts, not a bag of loose parts due to our incomplete step assy emission code.


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 76ab121..075f811 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -225,8 +225,8 @@ step_do_export (HID_Attr_Val * options)
   } while ((piece = piece->f) != board_outline);
   poly_Free (&board_outline);
 
-//  object3d_list_export_to_step_part (board_outline_list, temp_pcb_filename);
-  object3d_list_export_to_step_assy (board_outline_list, temp_pcb_filename);
+  object3d_list_export_to_step_part (board_outline_list, temp_pcb_filename);
+//  object3d_list_export_to_step_assy (board_outline_list, temp_pcb_filename);
   g_list_free_full (board_outline_list, (GDestroyNotify)destroy_object3d);
 
   if (1) {
